runtime/doc/lua.txt

```diff
@@ -1634,6 +1634,17 @@ tbl_flatten({t})                                           *vim.tbl_flatten()*
                 See also: ~
                     From https://github.com/premake/premake-core/blob/master/src/base/table.lua
 
+tbl_get({o}, {keys})                                           *vim.tbl_get()*
+                Index into a table using a list of keys. Return `nil` if the
+                key does not exist.
```

Comment:
An example here would be good I think since it's not super obvious what this does just from the description.

Reply:
Better?